plugins/stubcontent/class.stubcontent.plugin.php

```diff
@@ -201,7 +201,7 @@ public function addStubContent($type) {
     public function insertContent($content) {
 
         // Don't affect installed forums
-        if (c('Garden.Installed', false) === true) {
+        if (c('Garden.Installed', false) == true) {
             return;
         }
```

Comment:
I see why you are doing this (==) because on infrastructure we use "1" and "0" but it feels like a step in the wrong direction.